### ISSUE TITLE
"I apologize for the inconvenience, but I would like to confirm that inclusion_3000_exclusion_5000 means the vessel is located anywhere within a radius of 3 to 5 kilometers from the hydrophone. Similarly, inclusion_4000_exclusion_6000 indicates a range of 4 to 6 kilometers, while inclusion_2000_exclusion_4000 suggests a radius of 2 to 4 kilometers. However, in the document, 2000_exclusion_4000 seems to indicate a range of 0 to 2 kilometers.

### ISSUE THREAD

**Issue by jinanqi** [lucascesarfd/underwater_snd#7]
"I apologize for the inconvenience, but I would like to confirm that inclusion_3000_exclusion_5000 means the vessel is located anywhere within a radius of 3 to 5 kilometers from the hydrophone. Similarly, inclusion_4000_exclusion_6000 indicates a range of 4 to 6 kilometers, while inclusion_2000_exclusion_4000 suggests a radius of 2 to 4 kilometers. However, in the document, 2000_exclusion_4000 seems to indicate a range of 0 to 2 kilometers.

**Comment by psmskelton** [comment]
@jinanqi Not quite. The inclusion radius is where a single vessel, as much as can be guaranteed with available AIS data, is present within that radius. The exclusion radius means no other vessel, again as much as we can guarantee with available AIS data, is present within that radius:

inclusion_3000_exclusion_5000 = 1 vessel within 3000 metres, and no other vessels within 5000 metres.
inclusion_4000_exclusion_6000  = 1 vessel within 4000 metres, and no other vessels within 6000 metres.

Hope that clarifies it.
